roles/openshift_health_checker/meta/main.yml

```diff
@@ -2,3 +2,4 @@
 dependencies:
   - role: openshift_facts
   - role: openshift_repos
+  - role: openshift_version
```

Comment:
I need the `openshift_image_tag` task_var to be set in order to determine which version of ovs to check for